setup.py

```diff
@@ -0,0 +1,20 @@
+from Cython.Build import cythonize
+from setuptools import Extension, setup
+
+# Define Cython extension module
+extensions = [Extension("merge_ts.merge_ts_loop", ["merge_ts/merge_ts_loop.pyx"])]
```

Comment:
Is this part and the entry_points reasons this has to be it's own repository? Surely this can still fit in workflow? Not sure what else would be added to this repo apart from a single script, just feels a bit too much getting another repository to get workflow working just for a single script.

Reply:
Sung thought it made sense to put in it's own repo when I discussed how best to incorporate it with Cython.